hex_blueprints/check_run_status.py

```diff
@@ -0,0 +1,128 @@
+from dataclasses import dataclass
+import requests
+import shipyard_utils as shipyard
+import argparse
+import sys
+import re
+
+try:
+    import exit_codes as ec
+except BaseException:
+    from . import exit_codes as ec
+@dataclass
```

Comment:
What's a dataclass?

Reply:
A dataclass is essentially a methodless class (like a struct in other languages). It has attributes, but not methods [](edited-with-Axolo)